examples/crt/main.go

```
@@ -0,0 +1,305 @@
+func (g *Game) Draw(screen *ebiten.Image) {
+	// Write your game's rendering.
+	bgopts := &ebiten.DrawImageOptions{}
+	g.Offscreen.DrawImage(bgImage, bgopts)
+	i, _ := getPlayerIndex(g.Objects)
+	p := g.Objects[i]
+	DrawOffsetX := -(p.VX / 640) * 200
+	DrawOffsetY := -(p.VY / 480) * 50
+	for z := 0; z < 3; z++ {
+		for _, obj := range g.Objects {
+			if !obj.Hidden {
+				if obj.Z == z {
+					obj.Draw(g.Offscreen, DrawOffsetX, DrawOffsetY)
+				}
+			}
+		}
+	}
+	xJitter, yJitter := math.Sincos(float64(g.Time / 10))
+	text.Draw(g.Offscreen, "COLLECT EBITEN!", mainFont, int(DrawOffsetX+340+xJitter*2), int(DrawOffsetY+40+yJitter*2), color.White)
+	text.Draw(g.Offscreen, fmt.Sprintf("%d", g.Score), mainFont, int(DrawOffsetX+50+xJitter*2), int(DrawOffsetY+40+yJitter*2), color.White)
+	opts := &ebiten.DrawImageOptions{}
+	opts.GeoM.Translate(DrawOffsetX+xJitter*2, DrawOffsetY+yJitter*2)
+	g.Offscreen.DrawImage(ebitenImage, opts)
+	// These last few lines of the draw loop are of concern if you are looking to execute the crt shader.
+	op := &ebiten.DrawRectShaderOptions{}
+	op.Uniforms = map[string]interface{}{
+		"Time": float32(g.Time) / 60,
+	}
+	op.Images[0] = g.Offscreen
+	screen.DrawRectShader(640, 480, ntscShader, op)
+	g.Offscreen.Clear()
+}
+
+// Layout takes the outside size (e.g., the window size) and returns the (logical) screen size.
+// If you don't have to adjust the screen size with the outside size, just return a fixed size.
+func (g *Game) Layout(outsideWidth, outsideHeight int) (screenWidth, screenHeight int) {
+	return 640, 480
+}
+
+var (
+	gopherImage *ebiten.Image
+	floorImage  *ebiten.Image
+	ebitenImage *ebiten.Image
+	bgImage     *ebiten.Image
+	ntscShader  *ebiten.Shader
+	mainFont    font.Face
+)
+
+func main() {
+	rand.Seed(time.Now().UnixNano())
+	tt, err := opentype.Parse(fonts.PressStart2P_ttf)
+	if err != nil {
+		log.Fatal(err)
+	}
+	const dpi = 72
+	mainFont, _ = opentype.NewFace(tt, &opentype.FaceOptions{
```

Do not ignore the error.